scripts/generator/templates/template-vaadin-spring-bom.xml

```diff
@@ -78,6 +78,11 @@
                 <artifactId>vaadin-prod-bundle</artifactId>
                 <version>${project.version}</version>
             </dependency>
+            <dependency>
```

Comment:
This already has a core dependency. What does this add?

Reply:
This is needed here to tell bom the version of the vaadin-core-components.